.NET/Microsoft.Recognizers.Text.Number/Utilities/PatternUtil.cs

```diff
@@ -0,0 +1,285 @@
+﻿using Microsoft.Recognizers.Text.Number.Chinese;
+using Microsoft.Recognizers.Text.Number.Japanese;
+using System.Text;
+
+namespace Microsoft.Recognizers.Text.Number
+{
+    public class PatternUtil
```

Comment:
I don't think these methods should be here. They should be in a base CJK parser class, called from the Chinese and Japanese parsers, and use the configs in the parsers. Similar to how it works for English and other "related" languages.